modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_activation-keys.scss

```diff
@@ -0,0 +1,20 @@
+.cp-activation-keys-drop-down-item {
+	&:hover {
+		background-color: var(--color-action-primary-hover-lighten);
+		color: var(--color-action-primary-hover) !important;
+
+		svg {
+			color: var(--color-action-primary-hover) !important;
+			opacity: 0.5;
+		}
+	}
+}
```

Comment:
I don't know if this make sense, but for me it would be better like that:

```
.cp-activation-keys-drop-down-item :hover {
		background-color: var(--color-action-primary-hover-lighten);
		color: var(--color-action-primary-hover) !important;

		svg {
			color: var(--color-action-primary-hover) !important;
			opacity: 0.5;
		}
}
```

What you think about?

Reply:
Yeah! Agreed! I changed this! Thank you!